docs/example/README.md

```diff
@@ -0,0 +1,96 @@
+# Example of htsget-elsa process
+
+The following shows how to demonstrate htsget-elsa returning queries based on Elsa manifests.
+
+### Sharing with htsget enabled in Elsa
+1. Enable the htsget sharing on a release with some restrictions.
+2. Grab the JWT token from the data portal:
+   ```bash
+   export JWT_TOKEN=<JWT_TOKEN>
+   ```
+3. Query htsget for an allowed region:
+   ```bash
+   curl -H "Authorization: ${JWT_TOKEN}" "https://htsget-elsa.dev.umccr.org/reads/R001/8AE43A8E4C8111EE84492BBD28BC6E2F?referenceName=20&start=50888919&end=50931436"
```

Comment:
can we make the instructions specific as to what release restrictions are needed to make these examples work? (i..e for the autism restriction - it encompasses gene XYZ).

Also some guide as to where the magic strings in this URL can be derived. One is the release key - the other needs to be found in the manifest..

Reply:
i.e. should there be a "Download TSV" in the htsget panel inside Elsa? That is a basic TSV with all the magic strings mapped to ids?

Reply:
Yes, I don't think there's a way to get the htsget manifest without downloading it from the S3 bucket. A "Download TSV" button would probably be a good idea, but I believe that the htsget manifest currently uses hard-coded values anyway.

Reply:
I think it hashes the urls or sample ids probably.. so consistent - but not hard-coded